packages/desktop-client/package.json

```diff
@@ -53,7 +59,6 @@
     "postcss-preset-env": "6.3.1",
     "postcss-safe-parser": "4.0.1",
     "prop-types": "15.6.0",
-    "react": "16.13.1",
```

Comment:
The previous sorting was actually correct here.

```
mkdir test && cd test
yarn init
yarn add react
yarn add react-redux
cat package.json
```

```
{
  "name": "test",
  "version": "1.0.0",
  "main": "index.js",
  "license": "MIT",
  "dependencies": {
    "react": "^18.2.0",
    "react-redux": "^8.0.5"
  }
}
```

Reply:
Fixed! TIL VS Code’s “sort lines” function works differently (probably because of the quotes)